CIP-????/README.md

```diff
@@ -0,0 +1,311 @@
+
+The solution described in this CIP achieves the goal of ensuring the uniqueness of NFTs minted on the Cardano blockchain by using an off-chain data structure to track the minting and burning of NFTs, and storing the hash of this data structure together with the proofs on the blockchain. This allows the on-chain code to verify the uniqueness of an NFT at minting time, without requiring the entire data structure to be stored on the blockchain.
+
+Also, it allows anyone to independently build the off-chain data tree by retrieving on-chain minting/burning transactions. Therefore, there is no need for centralized solutions to validate the integrity of the tree, promoting decentralization and transparency within the Cardano ecosystem. Though, any - even un-trusted or adversarial -  3rd-party services can be used to be queried for a proof.
+
+The use of IntegriTree as the off-chain data structure allows for efficient and secure tracking of the minting and burning of NFTs. IntegriTree is a new accumulator scheme that leverages a complete binary hash tree for compact accumulation and dynamic updates. 
+
+The tree is used to accumulate the elements in the set, and to provide proofs. 
+The values are represented as open-interval pairs of consecutive elements with each node in the tree. IntegriTree offers proofs for membership, non-membership, addition, and deletion, with a proof size that scales logarithmically with the number of elements.
+
+The use of an accumulator to represent the state of the IntegriTree allows for the updated state to be verified and recorded on the blockchain in a single transaction. This simplifies the process of updating the state, and reduces the risk of errors or disputes.
+
+
+## Path to Active
+
+### Acceptance Criteria
+The acceptance criteria for this CIP to become active are as follows:
+
+- The solution described in the CIP has been implemented and tested on the Cardano blockchain.
+- The implementation has been reviewed and approved by the Cardano Foundation and IOHK.
+- The community has had sufficient time to review and provide feedback on the CIP and the implementation.
+- Any concerns or issues raised during the review and testing process have been addressed and resolved.
+
+## Implementation Plan
+
+
+The implementation plan for this CIP is as follows:
+
+- The solution described in the CIP will be implemented and tested - on the Cardano blockchain by the proposer of the CIP.
+- The proposer will submit the implementation for review and - approval by the Cardano Foundation and IOHK.
+- The proposer will make the implementation and the CIP available for review and feedback by the community.
+- The proposer will address and resolve any concerns or issues raised during the review and testing process.
+- Once the acceptance criteria have been met, the proposer will submit a pull request to the Cardano Improvement Proposals repository to update the status of the CIP to active.
+
+## Optional sections
+
+### Versioning
```

Comment:
```suggestion
## Versioning
```
The _Versioning_ section is much appreciated, although "Optional sections" is not actually a section 😅